### DISCUSSION TITLE
FirebaseException getToken

### DISCUSSION THREAD

**Discussion by gerritwitkamp** [firebase/flutterfire#9605] [Q&A]
Hey guys, my Flutter-app is using firebase_messaging for receiving push notifications. In my APM (Sentry) the following error is getting logged:

_FirebaseException method_channel_messaging.dart in MethodChannelFirebaseMessaging.getToken
[firebase_messaging/unknown] The network connection was lost._

Anyone an idea what could cause this and how to fix it? Seems like it only applies to Apple devices.

Thanks in advance.

**Comment by gerritwitkamp** [comment]
Anyone?

**Comment by tvanhuu** [comment]
Same issue please fix 👍

**Comment by dmsrkfcl15** [comment]
Same issue in version 14.6.1

**Comment by yosef1408** [comment]
facing the same issue

**Comment by EgHubs** [comment]
Same here!
